frontend/src/app/navbar/navbar.component.html

```diff
@@ -1,22 +1,10 @@
-<nav class="navbar navbar-light bg-light" style="margin-bottom: 50px;">
-  <a class="navbar-brand" href="#">
-    <img src="../../assets/logo.png" width="250" height="115" alt="Bulwark Logo" /></a>
-  <div class="mx-auto">
-    <div *ngIf="loaderService.isLoading | async" role="status">
-      <p-progressSpinner [style]="{width: '75px', height: '75px'}"></p-progressSpinner>
-    </div>
-  </div>
-  <div class="nav-item dropdown" *ngIf="loggedIn$ | async as loggedIn">
-    <a class="nav-link dropdown-toggle" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true"
-      aria-expanded="false">
-      Menu
+<div *ngIf="loaderService.isLoading | async" role="status">
```

Comment:
Currently, the loading spinner is rendering on top of the loading spinner causing it to push the menu bar down with every API request.  The loading spinner should display in the center of the menu bar and should not cause the menu bar to move.

Reply:
hi @Whamo12, what do you think about moving the loading bar to cover the whole page using something like [ng-block-ui](https://github.com/kuuurt13/ng-block-ui)?